packages/web/src/scss/components/Collapse/index.html

```diff
@@ -152,21 +150,65 @@ <h2 class="docs-Heading">Collapse with hide trigger when expanded </h2>
     >
       ...more
```

Comment:
(There should be a correct ellipsis `…` character. Feel free to fix it or ignore it.)

```suggestion
      … more
```